pandas/tests/reshape/test_reshape.py

```diff
@@ -608,6 +608,23 @@ def test_get_dummies_all_sparse(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("values", ["baz", "zoo"])
```

Comment:
are these two values testing meaningfully distinct cases?

Reply:
I thought of adding 2 different values just for the check but they are testing the same case. I will remove it.

Reply:
Only one now!